### PR TITLE
Spell doc 2024-10-17

### PR DESCRIPTION
**Pull request by 0xldr**

### CODE REVIEW

**Comment by 0xldr** [comment]
TODO:

- [ ] Finalize parameters
- [ ] Add remaining missing links
- [ ] Add doc links
- [ ] Add Spark spell address

File: governance/votes/Executive vote - October 17, 2024.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+
+If this executive proposal passes, the following **actions** will occur within the Sky Protocol:
+- The Lockstake Engine will be initialized. This requires a number of subactions:
+  - Activate a new MKROSM.
+  - Activate a new VoteDelegateFactory contract.
+  - Initialize the Lockstake Engine farm in the Flapper.
+  - Update Smart Burn Engine parameters.
+  - Initialize the Lockstake Engine and borrowing facility.
+  - The Airdrop Multisig for Early Bird Rewards will be funded with newly minted SKY tokens.
+- Debt Ceilings for the now defunct Monetalis vaults will be set to 0.
```
Comment (MakerJanSky) [inline governance/votes/Executive vote - October 17, 2024.md]:
```suggestion
- Debt Ceilings for the now defunct RWA vaults operated by Monetalis will be set to 0.
```

File: governance/votes/Executive vote - October 17, 2024.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+  - `step`: **60 seconds**.
+- REWARDS_LSMKR_USDS will be added to the Lockstake Engine as the first farm.
+- The LockstakeEngine will be authorized to acccess the `vat`.
+- The LockstakeClipper will be authorized to access the `vat` and the LockstakeEngine.
+- CLIPPER_MOM, MCD_DOG and MCD_END will be authorized to access LockstakeClipper.
+- New chainlog keys LOCKSTAKE_MKR, LOCKSTAKE_ENGINE, LOCKSTAKE_CLIP and LOCKSTAKE_CLIP_CALC will be added.
+
+#### Airdrop Multisig Funding
+
+To facilitate the payment of [Early Bird Rewards](https://sky-atlas.powerhouse.io/#A.5.2.1.1_Reward_System-99e76538-f850-4937-8092-d166cae5aff4|8d5a9e88cf49) **28,220,925.60 SKY** will be minted to a multisig controlled by the Accessibility Facilitators at [0x14D98650d46BF7679BBD05D4f615A1547C87Bf68](https://etherscan.io/address/0x14D98650d46BF7679BBD05D4f615A1547C87Bf68).
```
Comment (MakerJanSky) [inline governance/votes/Executive vote - October 17, 2024.md:154]:
```suggestion
To facilitate the payment of [Early Bird Rewards](https://sky-atlas.powerhouse.io/#A.5.2.1.1_Reward_System-99e76538-f850-4937-8092-d166cae5aff4|8d5a9e88cf49) **27,222,832.80 SKY** will be minted to a multisig controlled by the Accessibility Facilitators at [0x14D98650d46BF7679BBD05D4f615A1547C87Bf68](https://etherscan.io/address/0x14D98650d46BF7679BBD05D4f615A1547C87Bf68).
```

**Review by MakerJanSky — CHANGES_REQUESTED** [review]
First pass, formatting and presentation are well done. Minor changes, full checklist review to follow once final parameters are posted.

**Comment by 0xldr** [comment]
# Executive Creation

## Creation

- [ ] Sync Community Git
- [ ] Ensure you are working from the latest master branch
- [ ] Read Associated [Executive Contents sheet](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit?pli=1#gid=1593813984)
- [ ] Copy the executive template
- [ ] Update filename
- [ ] Update dates - exact time is not required and should be left at midnight.

- [ ] Ensure the title refers to the most impactful/important actions included in the proposal.
- [ ] Ensure that the summary refers to all actions included in the proposal.

- [ ] Ensure that the publisher (entity publishing the executive on the voting portal) of the executive is listed in the preamble (at the entity level).
- [ ] Ensure that the writers of the executive code are listed in the preamble (at the entity level).
- [ ] Ensure that the reviewers of the executive code are listed in the preamble (at the entity level).

- [ ] Ensure that the office-hours modifier is mentioned as part of the executive copy, if present in the executive spell code.
- [ ] Ensure that it is clearly stated which items are affected by the GSM Delay.
- [ ] Ensure that the GSM Delay period is clearly communicated.
- [ ] Ensure that the proposal expiry is explicitly noted, including the length of time for which the proposal is valid.

## Items

- [ ] Add each item to the proposal details section
- [ ] Order the items by judged impact/importance. If different from the executive contents, reorder the executive contents sheet so that they match.

- [ ] For each item
	- [ ] Ensure the executive summary contains a high-level overview of every action contained in the proposal.
	- [ ] Ensure that each item or set of items in the executive proposal has a corresponding entry in the proposal details section.
	- [ ] Ensure that each item in the copy contains a link to the reasoning for its inclusion.
	- [ ] Ensure that each item in the copy contains a link to the authorization for its inclusion.
	- [ ] Ensure that each item in the copy contains any links required for voters to determine the context surrounding its inclusion.
	- [ ] Ensure that the framing and tense used for each item is future-conditional. Ie: "IF this executive proposal passes, THEN this action will take place."
	- [ ] Ensure that hyperlinks to documentation and/or context are added to key parameters, terms, and contracts where reasonable and possible.
	- [ ] Ensure that Ethereum and IPFS addresses are hyperlinked such that it is easy to view their details.
	- [ ] Ensure that parties implicated in the proposal outcome related to this item are aware of the wording used.

## Self-Review

- [ ] Do any '$' parameter signifiers remain? (spell address is unknown at this point)
- [ ] Do the executive title, summary, and executive summary accurately reflect executive copy content?
- [ ] Read the executive copy in full, with the above checklists and general readability in mind. Make any required changes.
- [ ] Run the copy through a spelling and grammar checker - do not blindly accept suggested changes.
- [ ] Inform the reviewer that the executive copy is ready to review
- [ ] Commit changes, and create a PR to the Community GitHub repository.

## Submission

- [ ] Merge approved executive copy PR.
- [ ] Inform technical teams that the copy is ready and provide the link.
- [ ] Once technical teams have provided a spell address, add it to the executive copy.
- [ ] Update the proposals.json to set the active proposal on the portal.
- [ ] Refresh portal cache (DUX/Jetstream can provide password)
- [ ] Inform the community of the new executive proposal.

File: governance/votes/Executive vote - October 17, 2024.md [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+The Governance Facilitators, Sidestream, and Dewiz have placed an executive proposal into the voting system. MKR/SKY Holders should vote for this proposal if they support the following alterations to the Sky Protocol.
+
+If you are new to voting in the Sky Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **actions** will occur within the Sky Protocol:
+- The Seal Engine will be initialized. This requires a number of subactions:
```
Comment (CivicSage3) [inline governance/votes/Executive vote - October 17, 2024.md:19]:
```suggestion
- The Seal Engine will be initialized. This requires a number of sub-actions:
```

File: governance/votes/Executive vote - October 17, 2024.md [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+  - Activate a new MKROSM.
+  - Activate a new VoteDelegateFactory contract.
+  - Initialize the Seal Engine farm in the Flapper.
+  - Update Smart Burn Engine parameters.
+  - Initialize the Seal Engine and borrowing facility.
+  - The Multisig responsible for distributing Early Bird Rewards will be funded with newly minted SKY tokens.
```
Comment (CivicSage3) [inline governance/votes/Executive vote - October 17, 2024.md:25]:
```suggestion
  - Fund the Multisig responsible for distributing Early Bird Rewards with newly minted SKY tokens.
```
For the sake of consistency and ease of reading

File: governance/votes/Executive vote - October 17, 2024.md [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Sky Protocol.
+
+---
+
+## Proposal Details
+
+### Seal Engine Initialization
+
+- **Authorization**: [Governance Poll](https://vote.makerdao.com/polling/QmUm8Krq)
```
Comment (CivicSage3) [inline governance/votes/Executive vote - October 17, 2024.md:45]: @MakerJanSky this is missing in the Exec. Sheet

File: governance/votes/Executive vote - October 17, 2024.md [hunk 6]
```diff
@@ -0,0 +1,287 @@
...
+- **Proposal**: [Forum Post](https://forum.sky.money/t/atlas-weekly-cycle-edit-proposal-week-of-october-14-2024-01/25324)
+
+If this executive proposal passes, then the Seal Engine will be initialized through the following subactions.
```
Comment (CivicSage3) [inline governance/votes/Executive vote - October 17, 2024.md:48]:
```suggestion
If this executive proposal passes, then the Seal Engine will be initialized through the following sub-actions:
```

File: governance/votes/Executive vote - October 17, 2024.md [hunk 7]
```diff
@@ -0,0 +1,287 @@
...
+
+#### MKROSM Activation
+
+- The new MKROSM at [0x4F94e33D0D74CfF5Ca0D3a66F1A650628551C56b](https://etherscan.io/address/0x4F94e33D0D74CfF5Ca0D3a66F1A650628551C56b) will be authorized to read the MKR oracle price from the current PIP_MKR at [0xdbbe5e9b1daa91430cf0772fcebe53f6c6f137df](https://etherscan.io/address/0xdbbe5e9b1daa91430cf0772fcebe53f6c6f137df).
+- The new MKROSM at [0x4F94e33D0D74CfF5Ca0D3a66F1A650628551C56b](https://etherscan.io/address/0x4F94e33D0D74CfF5Ca0D3a66F1A650628551C56b) will replace the current PIP_MKR in the [Chainlog](https://chainlog.sky.money/).
+
+#### New VoteDelegateFactory Contract Activation
+
+- The current VOTE_DELEGATE_PROXY_FACTORY in the Chainlog](https://chainlog.sky.money/) at [0xd897f108670903d1d6070fcf818f9db3615af272](https://etherscan.io/address/0xd897f108670903d1d6070fcf818f9db3615af272) will be renamed to VOTE_DELEGATE_FACTORY_LEGACY.
```
Comment (CivicSage3) [inline governance/votes/Executive vote - October 17, 2024.md:57]:
```suggestion
- The current VOTE_DELEGATE_PROXY_FACTORY in the [Chainlog](https://chainlog.sky.money/) at [0xd897f108670903d1d6070fcf818f9db3615af272](https://etherscan.io/address/0xd897f108670903d1d6070fcf818f9db3615af272) will be renamed to VOTE_DELEGATE_FACTORY_LEGACY.
```

File: governance/votes/Executive vote - October 17, 2024.md [hunk 8]
```diff
@@ -0,0 +1,287 @@
...
+### Seal Engine Initialization
+
+- **Authorization**: [Governance Poll](https://vote.makerdao.com/polling/QmUm8Krq)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/atlas-weekly-cycle-edit-proposal-week-of-october-14-2024-01/25324)
+
+If this executive proposal passes, then the Seal Engine will be initialized through the following subactions.
+
+#### MKROSM Activation
+
+- The new MKROSM at [0x4F94e33D0D74CfF5Ca0D3a66F1A650628551C56b](https://etherscan.io/address/0x4F94e33D0D74CfF5Ca0D3a66F1A650628551C56b) will be authorized to read the MKR oracle price from the current PIP_MKR at [0xdbbe5e9b1daa91430cf0772fcebe53f6c6f137df](https://etherscan.io/address/0xdbbe5e9b1daa91430cf0772fcebe53f6c6f137df).
```
Comment (VoteWizard) [inline governance/votes/Executive vote - October 17, 2024.md:52]:
```suggestion
- The new MKROSM at [0x4F94e33D0D74CfF5Ca0D3a66F1A650628551C56b](https://etherscan.io/address/0x4F94e33D0D74CfF5Ca0D3a66F1A650628551C56b) will be whitelisted to read the MKR oracle price from the current PIP_MKR at [0xdbbe5e9b1daa91430cf0772fcebe53f6c6f137df](https://etherscan.io/address/0xdbbe5e9b1daa91430cf0772fcebe53f6c6f137df).
```
Based on Dewiz’s feedback, shouldn’t we be using the verb "whitelist" instead of "authorize" in these cases?

File: governance/votes/Executive vote - October 17, 2024.md (hunk 5 above)
Comment (MakerJanSky) [reply governance/votes/Executive vote - October 17, 2024.md:45]: Fixed. Thank you, CivicSage.

File: governance/votes/Executive vote - October 17, 2024.md [hunk 9]
```diff
@@ -0,0 +1,287 @@
...
+
+RWA014-A does not use the [Debt Ceiling Instant Access Module (DC-IAM)](https://sky-atlas.powerhouse.io#A.3.8.1.1.2.4_Debt_Ceiling_Instant_Access_Module_(DC_IAM)-071d42e3-8a21-4401-852e-0b52c49768bb|57eaf45219bea3b430c2).
+
+- Reduce the RWA014-A Debt Ceiling by 1.5 billion DAI from 1.5 billion DAI to **0 DAI**.
+
+#### Oracle Adjustments
+
+As part of the offboarding process, `tell` and `cull` will be called on the oracles for these vaults.
+
+Calling `tell` will initiate soft liquidation of these vaults and calling `cull` will write off their debt and set the oracle prices to 0. In reality, as neither of these vaults have associated debt the only change will be to set the reported prices of their oracles to 0.
```
Comment (CivicSage3) [inline governance/votes/Executive vote - October 17, 2024.md:180]:
```suggestion
Calling `tell` will initiate soft liquidation of these vaults and calling `cull` will write off their debt and set the oracle prices to 0. In reality, as neither of these vaults have associated debt, the only change will be to set the reported prices of their oracles to 0.
```

File: governance/votes/Executive vote - October 17, 2024.md [hunk 10]
```diff
@@ -0,0 +1,287 @@
...
+- StairstepExponentialDecrease calc contract will be configured with the following parameters:
+  - [`cut`](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.5.1.1_Auction_Price_Function_(cut)-b6e47d05-addf-4371-ad95-90582da7c0f1|57eaf45219bea3b430c2a59a6013): **0.99**.
+  - [`step`](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.5.1.2_Auction_Price_Function_(step)-7d58c961-43ff-4157-80aa-8a92f577ec12|57eaf45219bea3b430c2a59a6013): **60 seconds**.
+- REWARDS_LSMKR_USDS will be added to the Seal Engine as the first farm.
+- The LockstakeEngine contract will be authorized to access the `vat`.
+- The LockstakeClipper contract will be authorized to access the `vat` and the LockstakeEngine.
+- CLIPPER_MOM, MCD_DOG, and MCD_END will be authorized to access LockstakeClipper.
+- New [Chainlog](https://chainlog.sky.money) keys LOCKSTAKE_MKR, LOCKSTAKE_ENGINE, LOCKSTAKE_CLIP, and LOCKSTAKE_CLIP_CALC will be added.
+
+#### Early Bird Rewards Multisig Funding
```
Comment (VoteWizard) [inline governance/votes/Executive vote - October 17, 2024.md:152]:
```suggestion
### Early Bird Rewards Multisig Funding
```
Imo, this is not related to the Seal Engine Initialization

Comment (MakerJanSky) [reply governance/votes/Executive vote - October 17, 2024.md:152]: Fair assessment. I will change the Executive Summary to reflect this as well.

File: governance/votes/Executive vote - October 17, 2024.md [hunk 11]
```diff
@@ -0,0 +1,287 @@
...
+
+- **Authorization**: [Forum post](https://forum.sky.money/t/2024-10-17-expected-executive-contents-rwa-vault-changes/25323/2)
+- **Proposal**: [Forum post](https://forum.sky.money/t/2024-10-17-expected-executive-contents-rwa-vault-changes/25323)
+
+If this executive proposal passes, then the following Debt Ceiling changes for the now deprecated RWA Vaults operated by Monetalis will be carried out.
+
+#### Monetalis Clydesdale (RWA007-A)
+
+- RWA007-A will be removed from the [Debt Ceiling Instant Access Module (DC-IAM)](https://sky-atlas.powerhouse.io#A.3.8.1.1.2.4_Debt_Ceiling_Instant_Access_Module_(DC_IAM)-071d42e3-8a21-4401-852e-0b52c49768bb|57eaf45219bea3b430c2).
+- RWA007-A Debt Ceiling will be set to **0 DAI**.
```
Comment (VoteWizard) [inline governance/votes/Executive vote - October 17, 2024.md:168]:
```suggestion
- Reduce the RWA007-A Debt Ceiling by 3 billion DAI from 3 billion DAI to **0 DAI**.
```
This is more of a question than a suggestion. Why aren’t we using the same format as the Coinbase Custody Debt Ceiling change? Is it because Monetalis Clydesdale uses the DC-IAM?

Comment (MakerJanSky) [reply governance/votes/Executive vote - October 17, 2024.md:168]: You are correct. Monetalis is on the DC-IAM while Coinbase Custody is not.

Comment (MakerJanSky) [reply governance/votes/Executive vote - October 17, 2024.md:168]: And it has to be removed from the DC-IAM in order to set the DC to 0.

File: governance/votes/Executive vote - October 17, 2024.md [hunk 12]
```diff
@@ -0,0 +1,287 @@
+---
+title: Template - [Executive Vote] Seal Engine Initialization, RWA Vault Debt Ceiling Housekeeping, Pinwheel DAO Resolution, Aave-SparkLend Revenue Share Payment for Q3 2024, Spark Proxy Spell - October 17, 2024
+summary: Initialize the Seal Engine, reduce debt ceilings of deprecated RWA vaults, approve DAO Resolution for Pinwheel, transfer payment for the Aave-SparkLend Revenue Share for Q3 2024, execute a Spark Proxy Spell.
```
Comment (VoteWizard) [inline governance/votes/Executive vote - October 17, 2024.md:3]:
```suggestion
summary: Initialize the Seal Engine, fund the Multisig responsible for distributing Early Bird Rewards, reduce debt ceilings of deprecated RWA vaults, approve DAO Resolution for Pinwheel, transfer payment for the Aave-SparkLend Revenue Share for Q3 2024, execute a Spark Proxy Spell.
```
I would also add this here

Comment (MakerJanSky) [reply governance/votes/Executive vote - October 17, 2024.md:3]: Good catch.

File: governance/votes/Executive vote - October 17, 2024.md (hunk 11 above)
Comment (MakerJanSky) [reply governance/votes/Executive vote - October 17, 2024.md:168]: Would you like to resolve this one @VoteWizard , or would you still like it changed? We feel it is correct as is, but don't mind the suggestion if you prefer it.

**Comment by VoteWizard** [comment]
# Spell Coordination Sheet Review

## Checklist

- [x] Are all fields filled in?
- [x] Do contents match the discussion from the GovOps Meeting?
    - [x] Confirm all items were discussed at GovOps.
    - [x] Confirm no items from the discussion were omitted.
- [x] Are incomplete items (awaiting final figures) marked with a clear `TODO`?
- [x] If there is a SubDAO Proxy Spell:
    - [x] Is the proxy spell address listed in a "Trigger $SubDAO Proxy Spell at" line?
    - [x] Are SubDAO Spell contents seperated ([example](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit#gid=684547605&range=J26))?
- [x] Are the DAI transfers rounded to the nearest DAI (no decimal points)?
- [x] Are MKR transfers listed to exactly two decimal places?
- [x] Is the Checksum for DAI and MKR accurate?
- [x] Are Spell Items listed in order of importance for an average MKR voter?
- [x] Have all items been approved publicly by the appropriate party?

---


# Executive Document Review

## Content
- [x] Check the title refers to the most impactful/important actions included in the proposal.
- [x] Check that the summary refers to all actions included in the proposal.
- [x] Check that the publisher (entity publishing the executive on the voting portal) of the executive is listed in the preamble (at the entity level).
- [x] Check that the writers of the executive code are listed in the preamble (at the entity level).
- [x] Check that the reviewers of the executive code are listed in the preamble (at the entity level).
- [x] Check that the office-hours modifier is mentioned as part of the executive copy, if present in the executive spell code.
- [x] Check that it is clearly stated which items are affected by the GSM Delay.
- [x] Check that the GSM Delay period is communicated.
- [x] Check that the proposal expiry is explicitly noted, including the length of time for which the proposal is valid.

## Items

- [x] Check each item is present in the proposal details section
- [x] Check items are ordered by judged impact/importance

- [x] For each item
	- [x] Check the executive summary contains a high-level overview of the item.
	- [x] Check that each item or set of items in the executive proposal has a corresponding entry in the proposal details section.
	- [x] Check that the copy contains a link to the reasoning for its inclusion.
	- [x] Check that the copy contains a link to the authorization for its inclusion.
	- [x] Check that the copy contains any links required for voters to determine the context surrounding its inclusion.
	- [x] Check that the framing and tense used for each item is future-conditional. Ie: "IF this executive proposal passes, THEN this action will take place."
	- [x] Check that hyperlinks to documentation and/or context are added to key parameters, terms, and contracts where reasonable and possible.
	- [x] Check that Ethereum and IPFS addresses are hyperlinked such that it is easy to view their details.
	- [x] Check that parties implicated in the proposal outcome related to this item are aware of the wording used.

## Final Checks

- [x] Read the executive copy in full, with the above checklists and general readability in mind. Raise any issues with the author.

**Review by VoteWizard — APPROVED** [review]
Looks great! Thanks for the thorough work

**Review by MakerJanSky — APPROVED** [review]
Truly a team effort. Nice work, all.

**Comment by MakerJanSky** [comment]
Confirming the hash of `0xcbbb4fa0c3bce6e8d97c46e0d4a7aba50d42b184a6f58c0f1b1cf2e0da849858` on our side.
